### PR TITLE
rename nanogpt_generate to nanogpt to also support train

### PR DESCRIPTION
**Pull request by xmfan**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #1916

Differential Revision: [D49505905](https://our.internmc.facebook.com/intern/diff/D49505905)

### CODE REVIEW

File: userbenchmark/dynamo/common.py [hunk 1]
```diff
@@ -257,6 +257,7 @@ class CI(NamedTuple):
     *CI_SKIP[CI("aot_eager", training=False, dynamic=True)],
     *CI_SKIP[CI("inductor", training=False)],
     "nanogpt_generate",  # Assertion `index out of bounds: 0 <= tmp0 < 64` failed.
```
Comment (xuzhao9) [inline userbenchmark/dynamo/common.py:259]: I guess we need to remove `nanogpt_generate` after https://github.com/pytorch/benchmark/pull/1911 lands?

Comment (xmfan) [reply userbenchmark/dynamo/common.py:259]: that's right, I will remove it in #1911

**Comment by xmfan** [comment]
@xmfan has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D49505905).

**Comment by xmfan** [comment]
I don't actually need this change, we can do it within #1911

**Pull request closed without merging**